pkg/scaling/resolver/scale_resolvers.go

```diff
@@ -616,6 +631,18 @@ func resolveAuthSecret(ctx context.Context, client client.Client, logger logr.Lo
 	return string(result)
 }
 
+func readAuthParamsFromFile(filePath string) (map[string]string, error) {
+	data, err := os.ReadFile(filePath)
```

Comment:
the `filePath` should be treated not as an absolute path but instead relative path under the configured `authRootPath` parameter and only work if it's configured, otherwise throw an error.

Reply:
Updated in the most recent commit, thanks!